taipy/core/taipy.py

```diff
@@ -548,6 +552,8 @@ def get_scenarios(
     else:
         scenarios = []
 
+    if created_start_time or created_end_time:
+        scenarios = scenario_manager._filter_by_creation_time(scenarios, created_start_time, created_end_time)
     if is_sorted:
         scenario_manager._sort_scenarios(scenarios, descending, sort_key)
     return scenarios
```

Comment:
Does it make sense to completely delegate the logic of this method `get_scenarios` to the Scenario_manager?

Reply:
I don't think so. We already have manager methods to handle different options, no complicated logic is handled here

Reply:
All right.